### PR TITLE
Feature/remove entry italy

### PR DESCRIPTION
**Pull request by nicola-95**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -16,7 +16,6 @@ buildscript {
         classpath Deps.androidx_navigation
         classpath Deps.hilt_plugin
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.5.0"
```
Comment (astagi) [inline build.gradle:19]: Same here. Dokka is required to build documentation @nicola-95

File: sdk/build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ plugins {
     id 'kotlin-kapt'
     id 'dagger.hilt.android.plugin'
     id 'jacoco'
-    id 'org.jetbrains.dokka'
+    id 'org.jetbrains.dokka' version '1.6.21'
```
Comment (astagi) [inline sdk/build.gradle:9]: ok maybe you wanted a pinned version, right?

**Review by astagi — COMMENTED** [review]
Pinned version of Dokka looks ok, I don't know why it's been removed in Android app, maybe there is useless

**Comment by nicola-95** [comment]
@astagi, a recent commit of the decoder repository (this [one](https://github.com/eu-digital-green-certificates/dgca-app-core-android/commit/30caf48f25bb32bd650e1b87d5f65bd76d0252f9) to be precise) caused the App + SDK projects to break up, so I had to make these edits. Neither the app nor the SDK can build without the modifications involved in these PRs.

**Comment by astagi** [comment]
Thanks for the explanation @nicola-95

**Comment by nicola-95** [comment]
> Thanks for the explanation @nicola-95

You are welcome! :smile: Thanks to you.

**Pull request merged**